api/models/Condition.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const validate = require('validate.js');
+
+module.exports = {
+  attributes: {
+    conditionName: Sequelize.STRING,
+    icdCode: Sequelize.STRING,
+    symptomsBegan: Sequelize.STRING,
+    clinicallyImpair: Sequelize.STRING,
+    conditionOutlook: Sequelize.INTEGER, // fk to support
+    conditionOutlookUnknown: Sequelize.TEXT,
+    conditionLast: Sequelize.INTEGER, // fk to support
```

Comment:
Note: Chatted with Dave. Right now this integer is being looked up from code, but this means that the "human value" of this data exists outside of the database and database migrations. This could result in some weird mismatches between the database, the codebase, and/or anything that might use the database independently.

This looks good for now, but will add in a Trello card to move the value lookup into the database.